hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieArchivedTimeline.java

```diff
@@ -95,7 +95,9 @@ public HoodieArchivedTimeline(HoodieTableMetaClient metaClient) {
    *
    * @deprecated
    */
-  public HoodieArchivedTimeline() {}
+  public HoodieArchivedTimeline() {
+
```

Comment:
this new line add can be reverted.

Reply:
checkstyle was showing an error.. i'll remove the new line but the closing `}` should be alone on a line.